core/domain/question_fetchers.py

```diff
@@ -86,6 +86,32 @@ def get_questions_and_skill_descriptions_by_skill_ids(
     return questions, grouped_skill_descriptions
 
 
+def get_question_by_id(
```

Comment:
I think you can just call get_questions_by_id() with a single-element list (or use that existing function directly where you need it).

Reply:
Ah yes, sorry -- I missed that you needed versions. I think with the new function you have below, you can probably drop this one, yeah.